raiden/transfer/channel.py

```diff
@@ -1745,11 +1755,16 @@ def handle_receive_lockedtransfer_light(
         channel_state.partner_state.balance_proof = mediated_transfer.balance_proof
         channel_state.partner_state.merkletree = merkletree
 
+        print("raiden/transfer/channel.py:1806 PARTNER STATE UPDATED WITH BP {}".format(channel_state.partner_state.balance_proof))
```

Comment:
do we absolutely need this `print`?

Reply:
No, i will remove it, it was used just for debugging